src/main/java/com/develhope/spring/vehicle/optionals/models/Stereo.java

```diff
@@ -0,0 +1,59 @@
+package com.develhope.spring.vehicle.optionals.models;
+
+import com.develhope.spring.vehicle.optionals.Optionals;
+import com.develhope.spring.vehicle.optionals.enums.TypeOfStereos;
+import jakarta.persistence.*;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+@Entity
+@Table(name = "stereos")
+public class Stereo implements Optionals {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
```

Comment:
Come diceva stamattina Antonio nella Demo, per gli Enum va inserita l'annotazione mi pare che si chiami EnumeretedValue, altrimenti non capisce il database cosa sia. In caso la annotazione non fosse questa controlla un attimo la documentazione. Inoltre questa colonna deve essere anche Unique.

Reply:
Me l'ero scordato, grazie

Reply:
aspetta, ma hai taggato uno nel commento! LOL